### ISSUE TITLE
build issues on opensuse

### ISSUE THREAD

**Issue by agriggio** [artpixls/ART#64]
**[Original report](https://bitbucket.org/agriggio/art/issues/64) by Paolo Stivanin (Bitbucket: [Paolo](https://bitbucket.org/Paolo), GitHub: [Paolo](https://github.com/Paolo)).**

----------------------------------------

Hello,

I’d like to package ART for openSUSE, but I’m getting some compilation errors I can’t get around:

‌

```
[    2s] -- The C compiler identification is GNU 9.3.1
[    2s] -- The CXX compiler identification is GNU 9.3.1
[    2s] -- Check for working C compiler: /usr/bin/cc
[    2s] -- Check for working C compiler: /usr/bin/cc - works
[    2s] -- Detecting C compiler ABI info
[    2s] -- Detecting C compiler ABI info - done
[    2s] -- Detecting C compile features
[    2s] -- Detecting C compile features - done
[    2s] -- Check for working CXX compiler: /usr/bin/c++
[    2s] -- Check for working CXX compiler: /usr/bin/c++ - works
[    2s] -- Detecting CXX compiler ABI info
[    3s] -- Detecting CXX compiler ABI info - done
[    3s] -- Detecting CXX compile features
[    3s] -- Detecting CXX compile features - done
[    3s] -- CMAKE_BUILD_TYPE: release
[    3s] -- Found PkgConfig: /usr/bin/pkg-config (found version "1.6.3") 
[    3s] -- Checking for module 'gtk+-3.0>=3.16'
[    3s] --   Found gtk+-3.0, version 3.24.18
[    3s] -- Checking for module 'gtkmm-3.0>=3.16'
[    3s] --   Found gtkmm-3.0, version 3.24.2
[    4s] -- Checking for module 'glib-2.0>=2.44'
[    4s] --   Found glib-2.0, version 2.62.6
[    4s] -- Checking for module 'glibmm-2.4>=2.44'
[    4s] --   Found glibmm-2.4, version 2.62.0
[    4s] -- Checking for module 'cairomm-1.0'
[    4s] --   Found cairomm-1.0, version 1.12.2
[    4s] -- Checking for module 'gio-2.0>=2.44'
[    4s] --   Found gio-2.0, version 2.62.6
[    4s] -- Checking for module 'giomm-2.4>=2.44'
[    4s] --   Found giomm-2.4, version 2.62.0
[    4s] -- Checking for module 'gthread-2.0>=2.44'
[    4s] --   Found gthread-2.0, version 2.62.6
[    4s] -- Checking for module 'gobject-2.0>=2.44'
[    4s] --   Found gobject-2.0, version 2.62.6
[    4s] -- Checking for module 'sigc++-2.0>=2.3.1'
[    4s] --   Found sigc++-2.0, version 2.10.3
[    4s] -- Checking for module 'lensfun>=0.2'
[    4s] --   Found lensfun, version 0.3.2.0
[    4s] -- Checking for module 'librsvg-2.0>=2.40'
[    4s] --   Found librsvg-2.0, version 2.46.4
[    4s] -- Checking for module 'exiv2>=0.24'
[    4s] --   Found exiv2, version 0.27.2
[    4s] -- Checking for module 'lcms2>=2.6'
[    4s] --   Found lcms2, version 2.9
[    5s] -- Checking for module 'expat>=2.1'
[    5s] --   Found expat, version 2.2.9
[    5s] -- Checking for module 'fftw3f'
[    5s] --   Found fftw3f, version 3.3.8
[    5s] -- Checking for module 'libtiff-4>=4.0.4'
[    5s] --   Found libtiff-4, version 4.1.0
[    5s] -- Found JPEG: /usr/lib64/libjpeg.so (found version "80") 
[    5s] -- Found ZLIB: /usr/lib64/libz.so (found version "1.2.11") 
[    5s] -- Found PNG: /usr/lib64/libpng.so (found version "1.6.37") 
[    5s] -- Checking for module 'libcanberra-gtk3'
[    5s] --   Found libcanberra-gtk3, version 0.30
[    5s] -- Binutils version detected as less than 2.29 - setting CMake parameters to enable LTO linking:
[    5s]     CMAKE_AR="/usr/bin/gcc-ar"
[    5s]     CMAKE_RANLIB="/usr/bin/gcc-ranlib"
[    6s] -- Found OpenMP_C: -fopenmp (found version "4.5") 
[    6s] -- Found OpenMP_CXX: -fopenmp (found version "4.5") 
[    6s] -- Found OpenMP: TRUE (found version "4.5")  
[    6s] -- Performing Test _fftw3f_multithread
[    6s] -- Performing Test _fftw3f_multithread - Failed
[    6s] -- Performing Test LENSFUN_HAS_LOAD_DIRECTORY
[    6s] -- Performing Test LENSFUN_HAS_LOAD_DIRECTORY - Success
[    6s] -- using tcmalloc library in /usr/lib64/libtcmalloc.so
[    6s] -- Configuring done
[    6s] CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
[    6s] Please set them or make sure they are set and tested correctly in the CMake files:
[    6s] _el
[    6s]     linked by target "rtengine" in directory /home/abuild/rpmbuild/BUILD/ART-1.2/rtengine
[    6s]     linked by target "rth-cli" in directory /home/abuild/rpmbuild/BUILD/ART-1.2/rtgui
[    6s]     linked by target "rth" in directory /home/abuild/rpmbuild/BUILD/ART-1.2/rtgui
[    6s] 
[    6s] -- Generating done
[    6s] CMake Warning:
[    6s]   Manually-specified variables were not used by the project:
[    6s] 
[    6s]     BUILD_STATIC_LIBS
[    6s]     CMAKE_Fortran_FLAGS
[    6s]     CMAKE_INSTALL_DO_STRIP
[    6s]     CMAKE_MODULES_INSTALL_DIR
[    6s]     INCLUDE_INSTALL_DIR
[    6s]     LIB_INSTALL_DIR
[    6s]     LIB_SUFFIX
[    6s]     SHARE_INSTALL_PREFIX
[    6s]     SYSCONF_INSTALL_DIR
[    6s] 
[    6s] 
[    6s] CMake Generate step failed.  Build files cannot be regenerated correctly.
[    6s] error: Bad exit status from /var/tmp/rpm-tmp.ibRSEj (%build)
```

What am I missing here?

Thanks!

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

Hi, can you try the attached patch? Thanks!

**Comment by agriggio** [comment]
**Original comment by Paolo Stivanin (Bitbucket: [Paolo](https://bitbucket.org/Paolo), GitHub: [Paolo](https://github.com/Paolo)).**

----------------------------------------

that was fast :smiley: I’m on it, thanks

**Comment by agriggio** [comment]
**Original comment by Paolo Stivanin (Bitbucket: [Paolo](https://bitbucket.org/Paolo), GitHub: [Paolo](https://github.com/Paolo)).**

----------------------------------------

nope, didn’t work:

```
[    1s] Patch #1 (fix-cmake.patch):
[    1s] + /usr/bin/patch --no-backup-if-mismatch -p1 --fuzz=0
[    1s] patching file CMakeLists.txt
[    1s] Hunk #1 succeeded at 340 (offset -2 lines).
[    1s] Hunk #2 succeeded at 566 (offset -2 lines).
```

and

```
[    6s] -- Found OpenMP: TRUE (found version "4.5")  
[    6s] -- Performing Test _fftw3f_multithread
[    6s] -- Performing Test _fftw3f_multithread - Failed
[    6s] -- searching for library lensfun
[    6s] --   result: /usr/lib64/liblensfun.so
[    6s] -- Performing Test LENSFUN_HAS_LOAD_DIRECTORY
[    6s] -- Performing Test LENSFUN_HAS_LOAD_DIRECTORY - Success
[    6s] -- using tcmalloc library in /usr/lib64/libtcmalloc.so
[    6s] -- Configuring done
[    6s] CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
[    6s] Please set them or make sure they are set and tested correctly in the CMake files:
[    6s] _el
[    6s]     linked by target "rtengine" in directory /home/abuild/rpmbuild/BUILD/ART-1.2/rtengine
[    6s]     linked by target "rth-cli" in directory /home/abuild/rpmbuild/BUILD/ART-1.2/rtgui
[    6s]     linked by target "rth" in directory /home/abuild/rpmbuild/BUILD/ART-1.2/rtgui
[    6s] 
[    6s] -- Generating done
[    6s] CMake Warning:
[    6s]   Manually-specified variables were not used by the project:
[    6s] 
[    6s]     BUILD_STATIC_LIBS
[    6s]     CMAKE_Fortran_FLAGS
[    6s]     CMAKE_INSTALL_DO_STRIP
[    6s]     CMAKE_MODULES_INSTALL_DIR
[    6s]     INCLUDE_INSTALL_DIR
[    6s]     LIB_INSTALL_DIR
[    6s]     LIB_SUFFIX
[    6s]     SHARE_INSTALL_PREFIX
[    6s]     SYSCONF_INSTALL_DIR
[    6s] 
[    6s] 
[    6s] CMake Generate step failed.  Build files cannot be regenerated correctly.
[    6s] error: Bad exit status from /var/tmp/rpm-tmp.OCD80v (%build)
```

‌

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

can you post the full log please? thanks! It seems it’s having troubles finding exiv2, but I’d like to see where it is searching…

**Comment by agriggio** [comment]
**Original comment by Paolo Stivanin (Bitbucket: [Paolo](https://bitbucket.org/Paolo), GitHub: [Paolo](https://github.com/Paolo)).**

----------------------------------------

```
uilding ART.spec for openSUSE_Tumbleweed/x86_64
Use local '/home/pstivanin/Documents/obs/home:polslinux:ART/ART/.osc/_buildinfo-openSUSE_Tumbleweed-x86_64.xml' file as buildinfo
Use local '/home/pstivanin/Documents/obs/home:polslinux:ART/ART/.osc/_buildconfig-openSUSE_Tumbleweed-x86_64' file as buildconfig
Updating cache of required packages
0.0% cache miss. 417/417 dependencies cached.

Skipping verification of package signatures
Writing build configuration
Running build
logging output to /var/tmp/build-root/openSUSE_Tumbleweed-x86_64/.build.log...
[    0s] Memory limit set to 43426724KB
[    0s] Using BUILD_ROOT=/var/tmp/build-root/openSUSE_Tumbleweed-x86_64
[    0s] Using BUILD_ARCH=x86_64:i686:i586:i486:i386
[    0s] 
[    0s] 
[    0s] suse-ws.fritz.box started "build ART.spec" at Fri Apr 24 10:11:17 UTC 2020.
[    0s] 
[    0s] 
[    0s] processing recipe /home/pstivanin/Documents/obs/home:polslinux:ART/ART/ART.spec ...
[    0s] running changelog2spec --target rpm --file /home/pstivanin/Documents/obs/home:polslinux:ART/ART/ART.spec
[    1s] cp: -r not specified; omitting directory '/home/pstivanin/Documents/obs/home:polslinux:ART/ART/ART-1.2'
[    1s] cp: -r not specified; omitting directory '/home/pstivanin/Documents/obs/home:polslinux:ART/ART/ART-1.2.orig'
[    1s] -----------------------------------------------------------------
[    1s] I have the following modifications for ART.spec:
[    1s] -----------------------------------------------------------------
[    1s] ----- building ART.spec (user abuild)
[    1s] -----------------------------------------------------------------
[    1s] -----------------------------------------------------------------
[    1s] + exec rpmbuild -ba --define '_srcdefattr (-,root,root)' --nosignature /home/abuild/rpmbuild/SOURCES/ART.spec
[    1s] setting SOURCE_DATE_EPOCH=1587686400
[    1s] Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.FHRp6d
[    1s] + umask 022
[    1s] + cd /home/abuild/rpmbuild/BUILD
[    1s] + cd /home/abuild/rpmbuild/BUILD
[    1s] + rm -rf ART-1.2
[    1s] + /usr/bin/tar -xof /home/abuild/rpmbuild/SOURCES/ART-1.2.tar.gz
[    1s] + STATUS=0
[    1s] + '[' 0 -ne 0 ']'
[    1s] + cd ART-1.2
[    1s] + /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
[    1s] + echo 'Patch #0 (add_releaseinfo.patch):'
[    1s] Patch #0 (add_releaseinfo.patch):
[    1s] + /usr/bin/patch --no-backup-if-mismatch -p0 --fuzz=0
[    1s] patching file a/ReleaseInfo.cmake
[    1s] + echo 'Patch #1 (fix-cmake.patch):'
[    1s] Patch #1 (fix-cmake.patch):
[    1s] + /usr/bin/patch --no-backup-if-mismatch -p1 --fuzz=0
[    1s] patching file CMakeLists.txt
[    1s] Hunk #1 succeeded at 340 (offset -2 lines).
[    1s] Hunk #2 succeeded at 566 (offset -2 lines).
[    1s] + RPM_EC=0
[    1s] ++ jobs -p
[    1s] + exit 0
[    1s] Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.6iLjbe
[    1s] + umask 022
[    1s] + cd /home/abuild/rpmbuild/BUILD
[    1s] + /usr/bin/rm -rf /home/abuild/rpmbuild/BUILDROOT/ART-1.2-0.x86_64
[    1s] ++ dirname /home/abuild/rpmbuild/BUILDROOT/ART-1.2-0.x86_64
[    1s] + /usr/bin/mkdir -p /home/abuild/rpmbuild/BUILDROOT
[    1s] + /usr/bin/mkdir /home/abuild/rpmbuild/BUILDROOT/ART-1.2-0.x86_64
[    1s] + cd ART-1.2
[    1s] + find . -name CMakeLists.txt -exec sed -i -re '/^[[:blank:]]*[sS][eE][tT][[:blank:]]*\([[:blank:]]*(CMAKE_BUILD_TYPE|CMAKE_COLOR_MAKEFILE|CMAKE_INSTALL_PREFIX|CMAKE_VERBOSE_MAKEFILE).*\)/{s/^/#IGNORE /}' '{}' +
[    1s] + mkdir -p build
[    1s] + cd build
[    1s] + /usr/bin/cmake /home/abuild/rpmbuild/BUILD/ART-1.2/. '-GUnix Makefiles' -DCMAKE_INSTALL_PREFIX:PATH=/usr -DINCLUDE_INSTALL_DIR:PATH=/usr/include -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DSYSCONF_INSTALL_DIR:PATH=/etc -DSHARE_INSTALL_PREFIX:PATH=/usr/share -DCMAKE_INSTALL_LIBDIR:PATH=/usr/lib64 -DCMAKE_BUILD_TYPE=RelWithDebInfo '-DCMAKE_C_FLAGS=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -DNDEBUG' '-DCMAKE_CXX_FLAGS=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -DNDEBUG' '-DCMAKE_Fortran_FLAGS=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -DNDEBUG' '-DCMAKE_EXE_LINKER_FLAGS=-flto=auto -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now' '-DCMAKE_MODULE_LINKER_FLAGS=-flto=auto -Wl,--as-needed' '-DCMAKE_SHARED_LINKER_FLAGS=-flto=auto -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now' -DLIB_SUFFIX=64 -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_STATIC_LIBS:BOOL=OFF -DCMAKE_COLOR_MAKEFILE:BOOL=OFF -DCMAKE_INSTALL_DO_STRIP:BOOL=OFF -DCMAKE_MODULES_INSTALL_DIR=/usr/share/cmake/Modules -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=release -DPROC_TARGET_NUMBER=2 -DENABLE_TCMALLOC=ON -DTCMALLOC_LIB_DIR=/usr/lib -DWITH_LTO=ON .
[    2s] -- The C compiler identification is GNU 9.3.1
[    2s] -- The CXX compiler identification is GNU 9.3.1
[    2s] -- Check for working C compiler: /usr/bin/cc
[    2s] -- Check for working C compiler: /usr/bin/cc - works
[    2s] -- Detecting C compiler ABI info
[    2s] -- Detecting C compiler ABI info - done
[    2s] -- Detecting C compile features
[    2s] -- Detecting C compile features - done
[    2s] -- Check for working CXX compiler: /usr/bin/c++
[    2s] -- Check for working CXX compiler: /usr/bin/c++ - works
[    2s] -- Detecting CXX compiler ABI info
[    3s] -- Detecting CXX compiler ABI info - done
[    3s] -- Detecting CXX compile features
[    3s] -- Detecting CXX compile features - done
[    3s] -- CMAKE_BUILD_TYPE: release
[    3s] -- Found PkgConfig: /usr/bin/pkg-config (found version "1.6.3") 
[    3s] -- Checking for module 'gtk+-3.0>=3.16'
[    3s] --   Found gtk+-3.0, version 3.24.18
[    3s] -- Checking for module 'gtkmm-3.0>=3.16'
[    3s] --   Found gtkmm-3.0, version 3.24.2
[    4s] -- Checking for module 'glib-2.0>=2.44'
[    4s] --   Found glib-2.0, version 2.62.6
[    4s] -- Checking for module 'glibmm-2.4>=2.44'
[    4s] --   Found glibmm-2.4, version 2.62.0
[    4s] -- Checking for module 'cairomm-1.0'
[    4s] --   Found cairomm-1.0, version 1.12.2
[    4s] -- Checking for module 'gio-2.0>=2.44'
[    4s] --   Found gio-2.0, version 2.62.6
[    4s] -- Checking for module 'giomm-2.4>=2.44'
[    4s] --   Found giomm-2.4, version 2.62.0
[    4s] -- Checking for module 'gthread-2.0>=2.44'
[    4s] --   Found gthread-2.0, version 2.62.6
[    4s] -- Checking for module 'gobject-2.0>=2.44'
[    4s] --   Found gobject-2.0, version 2.62.6
[    4s] -- Checking for module 'sigc++-2.0>=2.3.1'
[    4s] --   Found sigc++-2.0, version 2.10.3
[    4s] -- Checking for module 'lensfun>=0.2'
[    4s] --   Found lensfun, version 0.3.2.0
[    4s] -- Checking for module 'librsvg-2.0>=2.40'
[    4s] --   Found librsvg-2.0, version 2.46.4
[    5s] -- Checking for module 'exiv2>=0.24'
[    5s] --   Found exiv2, version 0.27.2
[    5s] -- searching for library exiv2 in /usr/usr/lib64
[    5s] --   result: _el-NOTFOUND
[    5s] -- Checking for module 'lcms2>=2.6'
[    5s] --   Found lcms2, version 2.9
[    5s] -- Checking for module 'expat>=2.1'
[    5s] --   Found expat, version 2.2.9
[    5s] -- Checking for module 'fftw3f'
[    5s] --   Found fftw3f, version 3.3.8
[    5s] -- Checking for module 'libtiff-4>=4.0.4'
[    5s] --   Found libtiff-4, version 4.1.0
[    5s] -- Found JPEG: /usr/lib64/libjpeg.so (found version "80") 
[    5s] -- Found ZLIB: /usr/lib64/libz.so (found version "1.2.11") 
[    5s] -- Found PNG: /usr/lib64/libpng.so (found version "1.6.37") 
[    5s] -- Checking for module 'libcanberra-gtk3'
[    5s] --   Found libcanberra-gtk3, version 0.30
[    5s] -- Binutils version detected as less than 2.29 - setting CMake parameters to enable LTO linking:
[    5s]     CMAKE_AR="/usr/bin/gcc-ar"
[    5s]     CMAKE_RANLIB="/usr/bin/gcc-ranlib"
[    6s] -- Found OpenMP_C: -fopenmp (found version "4.5") 
[    6s] -- Found OpenMP_CXX: -fopenmp (found version "4.5") 
[    6s] -- Found OpenMP: TRUE (found version "4.5")  
[    6s] -- Performing Test _fftw3f_multithread
[    6s] -- Performing Test _fftw3f_multithread - Failed
[    6s] -- searching for library lensfun
[    6s] --   result: /usr/lib64/liblensfun.so
[    6s] -- Performing Test LENSFUN_HAS_LOAD_DIRECTORY
[    6s] -- Performing Test LENSFUN_HAS_LOAD_DIRECTORY - Success
[    6s] -- using tcmalloc library in /usr/lib64/libtcmalloc.so
[    6s] -- Configuring done
[    6s] CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
[    6s] Please set them or make sure they are set and tested correctly in the CMake files:
[    6s] _el
[    6s]     linked by target "rtengine" in directory /home/abuild/rpmbuild/BUILD/ART-1.2/rtengine
[    6s]     linked by target "rth-cli" in directory /home/abuild/rpmbuild/BUILD/ART-1.2/rtgui
[    6s]     linked by target "rth" in directory /home/abuild/rpmbuild/BUILD/ART-1.2/rtgui
[    6s] 
[    6s] -- Generating done
[    6s] CMake Warning:
[    6s]   Manually-specified variables were not used by the project:
[    6s] 
[    6s]     BUILD_STATIC_LIBS
[    6s]     CMAKE_Fortran_FLAGS
[    6s]     CMAKE_INSTALL_DO_STRIP
[    6s]     CMAKE_MODULES_INSTALL_DIR
[    6s]     INCLUDE_INSTALL_DIR
[    6s]     LIB_INSTALL_DIR
[    6s]     LIB_SUFFIX
[    6s]     SHARE_INSTALL_PREFIX
[    6s]     SYSCONF_INSTALL_DIR
[    6s] 
[    7s] 
[    7s] CMake Generate step failed.  Build files cannot be regenerated correctly.
[    7s] error: Bad exit status from /var/tmp/rpm-tmp.6iLjbe (%build)
[    7s] 
[    7s] 
[    7s] RPM build errors:
[    7s]     Bad exit status from /var/tmp/rpm-tmp.6iLjbe (%build)
[    7s] 
[    7s] suse-ws.fritz.box failed "build ART.spec" at Fri Apr 24 10:11:24 UTC 2020.
[    7s] 

The buildroot was: /var/tmp/build-root/openSUSE_Tumbleweed-x86_64
```

‌

**Comment by agriggio** [comment]
**Original comment by Paolo Stivanin (Bitbucket: [Paolo](https://bitbucket.org/Paolo), GitHub: [Paolo](https://github.com/Paolo)).**

----------------------------------------

here the error:

```
[    5s] -- Checking for module 'exiv2>=0.24'
[    5s] --   Found exiv2, version 0.27.2
[    5s] -- searching for library exiv2 in /usr/usr/lib64
[    5s] --   result: _el-NOTFOUND
```

l missed it when looking at the terminal :slight_smile:

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

so here it is:

```
[    5s] -- Checking for module 'exiv2>=0.24'
[    5s] --   Found exiv2, version 0.27.2
[    5s] -- searching for library exiv2 in /usr/usr/lib64
[    5s] --   result: _el-NOTFOUND
```

can you check where your exiv2 library is installed? This path `/usr/usr/lib64` is quite suspicious… 

let me try a possible workaround

‌

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

Here's a new version of the patch -- let's see if this works

**Comment by agriggio** [comment]
**Original comment by Paolo Stivanin (Bitbucket: [Paolo](https://bitbucket.org/Paolo), GitHub: [Paolo](https://github.com/Paolo)).**

----------------------------------------

that worked fine :slight_smile: thanks a lot!

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

cmake: fix build issue on opensuse (exiv2 library not found)

Fixes #64
